src/gpgservice.py

```diff
@@ -0,0 +1,30 @@
+import gnupg  
+import json
+
+class GPGService:
+
+	def __init__(self):
+		self.gpg = gnupg.GPG(gnupghome = 'homegpg')
+		self.gpg.encoding = 'utf-8'
+
+	def generate_keys(self):
+		key = self.gpg.gen_key(self.gpg.gen_key_input(name_email = ''))
+		return key.fingerprint
+
+	def get_public_key(self, fingerprint):
+		return self.gpg.export_keys(fingerprint)
+
+	def add_conversation(self, public_key):
+		import_result = self.gpg.import_keys(public_key)
+		return import_result.fingerprints[0]
+
+	def encrypt_message(self, message, fingerprint):
+		message_params = {}
+		message_params['data'] = message
+		message_params['fingerprint'] = fingerprint
+		return self.gpg.encrypt(json.dumps(message_params), fingerprint)
```

Comment:
I think that `gpg.encrypt` returns `Crypt` object instead of something easily serializable. It should preferably return a string, bytes or dict.